corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -727,3 +728,23 @@ def test_report_module(self):
             app.create_suite(),
             "./detail[@id='reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.summary']",
         )
+
+        report_app_config._report.columns[0]['transform'] = {
+            'type': 'translation',
+            'translations': {
+                '1': [
+                    ['en', 'one'],
```

Comment:
any reason why this is a dict of lists of two-element lists instead of a dict of dicts? seems like the latter would be easier to work with (but maybe there's something about the ordering that's important?). if we're going to consider changing it'd be good to nail that down before anyone starts using it so we don't have to worry about migrating anything